STM32.json

```diff
@@ -0,0 +1,160 @@
+{
```

Comment:
Was this model file intended to be checked in? It looks like a copy of the other model in the PR but not in the expected location.

Reply:
Your are right, I have removed the json file from root folder.